runtime/src/execute_cost_table.rs

```diff
@@ -104,6 +133,16 @@ impl ExecuteCostTable {
         *timestamp = Self::micros_since_epoch();
     }
 
+    /// Simple calculation anomaly check, return True if calculation result is
+    /// LARGE_CHANGE_THRESHOLD times larger than input data itself;
+    fn is_large_change(input: u64, calculated_value: u64) -> bool {
+        const LARGE_CHANGE_THRESHOLD: u64 = 20;
+        const PERCENTAGE_OF_MAX_UNITS: f64 = 0.95;
+        let has_oversized_impact = input != 0 && calculated_value / input > LARGE_CHANGE_THRESHOLD;
+        let is_large_value = calculated_value as f64 / compute_budget::MAX_UNITS as f64 > PERCENTAGE_OF_MAX_UNITS;
+        has_oversized_impact || is_large_value
```

Comment:
Out of curiosity, what would it take to move the compute threshold by 20%? How much higher than the average would the newly executed transactions have to take?

Reply:
From that 7-day per-program-timing data I have for cndy, there are few cases like this: the average=156,266, a new cost is reported to be 7,614, if run the simple avg = (156,266 + 7,614)/2 = 81,940. This are the incidences might be interesting to capture. I haven't looked into cause of that yet, but the reported low units are not from errored execution (cause if it is from errored, it'd be brought back to prv avg).

Reply:
got it, so it's always `updated_cost = (old_cost + new_cost) / 2`, that's easier to reason about :). Ok so, that means if `new_cost >= 1.4 * old_cost`, or `new_cost <= .6 * old_cost`, this will fire right?

Reply:
Right now, it is (old+new)/2; EMA calls a slightly different method. Regardless the algo, if calculation result is significantly larger than input data, it'd be a signal for possible anomaly.